book/layout.md

```diff
@@ -281,17 +279,6 @@ class BlockLayout:
             child.layout()
 ```
 
-We also need to gather their `display_list` fields into a single
-array:
-
-``` {.python expected=False}
-class BlockLayout:
```

Comment:
This code blocks just looks stale, so good to remove?

Reply:
Yes, this implements that "stop-gap" described above.